lldb/test/API/python_api/type/TestTypeList.py

```diff
@@ -119,6 +119,15 @@ def test(self):
 
         self.assertEqual(task_type, task_head_pointee_type)
 
+        # Check whether we can find a nested type by name
+        name_type = task_type.FindNestedType("name")
+        self.assertTrue(name_type)
+        self.DebugSBType(name_type)
+
```

Comment:
Can you add a brief comment on what this is testing (as opposed to the assertion above).

Reply:
I pushed an update. Can you take a look again?

Reply:
thanks!